src/dolphin/install/installation.ts

```diff
@@ -176,11 +176,16 @@ export class DolphinInstallation {
   }
 
   private async _isOutOfDate(latestVersion: string): Promise<boolean> {
+    const dolphinVersion = await this.getDolphinVersion();
+    return lt(dolphinVersion, latestVersion);
```

Comment:
```suggestion
    const dolphinVersion = await this.getDolphinVersion();
    return dolphinVersion && lt(dolphinVersion, latestVersion);
```
If `dolphinVersion` is updated to be null we probably want to ensure it actually exists too.